ngraph_bridge/ngraph_executor.h

```diff
@@ -118,25 +121,12 @@ class NGraphExecutor {
   map<string, string> m_aot_execs;
 
   // serialized_ng_function, ng_executable, Output and Input Cache maps
-  std::unordered_map<std::string, std::shared_ptr<ngraph::runtime::Executable>>
-      m_ng_exec_map;
-  std::unordered_map<std::shared_ptr<ngraph::runtime::Executable>, std::string>
-      m_serialized_ng_function_map;
-
-  std::unordered_map<
-      std::shared_ptr<ngraph::runtime::Executable>,
-      std::vector<std::pair<void*, shared_ptr<ng::runtime::Tensor>>>>
-      m_ng_exec_input_cache_map;
-
-  std::unordered_map<
-      std::shared_ptr<ngraph::runtime::Executable>,
-      std::vector<std::pair<void*, shared_ptr<ng::runtime::Tensor>>>>
-      m_ng_exec_output_cache_map;
+  NgraphDataCache<std::string,
```

Comment:
Provide description. Key is the string (computed signature). what is the string that is part of the tuple which is the value?

Reply:
Done